src/commands/common/send-telemetry-command.js

```diff
@@ -33,7 +34,8 @@ class SendTelemetryCommand extends Command {
                     hostname: this.config.hostname,
                     operational_wallet: this.blockchainModuleManager.getPublicKey(),
                     management_wallet: this.blockchainModuleManager.getManagementKey(),
-                    triple_store: this.config.modules.tripleStore.defaultImplementation,
+                    triple_store: this.tripleStoreService.getRepositoryImplementations(),
```

Comment:
This is an object mapping triple store repository name (for example public-current) to their implementation name (for example "ot-blazegraph"). 
I think we should send implementationNames to telemetry, as we do for blockchain module

Reply:
@djordjekovac @NZT48  please confirm